content/developer/reference/cli.rst

```diff
@@ -669,18 +669,41 @@ Here is a sample file:
 Shell
 =====
 
-Odoo command-line also allows to launch odoo as a python console environment.
-This enables direct interaction with the :ref:`orm <reference/orm>` and its functionalities.
-
+The Odoo command line also allows launching Odoo as a Python console environment, enabling direct
+interaction with the :ref:`orm <reference/orm>` and its functionalities.
 
 .. code-block:: console
 
    $ odoo-bin shell
 
 .. option:: --shell-interface (ipython|ptpython|bpython|python)
 
-    Specify a preferred REPL to use in shell mode.
+   Specify a preferred REPL to use in shell mode. This shell is started with the `env` variable
+   already initialized to be able to access the ORM and other Odoo modules.
+
+.. example:: Example of shell usage
```

Comment:
I'd move the example before the `shell-interface` option; readers want to know how it works before diving into the details.